typedochtml.json

```diff
@@ -20,6 +20,6 @@
     "typedoc-plugin-ga"
   ],
   "themeColor": "#ff4e9f",
-  "customFooterHtml": "<p> Copyright © <strong>Arcana Technologies Ltd. </strong>All rights reserved</p>",
+  "customFooterHtml": "<p> Copyright © <2025 strong>Arcanaa Network. </strong>All rights reserved</p>",
```

Comment:
Typo in `Arcana`

Reply:
Yeah there is not just one but two typos, fixing.